landscape.yml

```diff
@@ -1534,6 +1534,12 @@ landscape:
             logo: ./hosted_logos/kubevirt.svg
             twitter: 'https://twitter.com/kubevirt'
             crunchbase: 'https://www.crunchbase.com/organization/kubevirt'
+          - item:
+            name: Minikube
+            homepage_url: 'https://github.com/kubernetes/minikube'
```

Comment:
We trigger whether it's open source based on the presence of the `repo_url`. You want to do:

```yml
homepage_url: 'https://kubernetes.io/docs/getting-started-guides/minikube/'
repo_url: 'https://github.com/kubernetes/minikube'
```

Reply:
Yup sorry I just added that. It should be in this commit already